airqo-platform/AirQo-frontend
# Set up GitGaurdian pre-commit hook

#### Summary of Changes (What does this PR do?)
- Adds ggshield pre-commit hook

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
- Update your local branch with `git pull origin` and switch to this branch with `git checkout --track origin/ft-secrets-security`.
- Follow the ggshield pre-commit [setup tutorial](https://docs.google.com/presentation/d/1vIoC-g9UmcAVarjEQgXpBK_bVbACaXLn3VzY8AWg6hU/edit#slide=id.g11b1ae9a83c_0_27) and attempt to commit a secret in your local `ft-secrets-security` branch. The fail message should list the file and/or line in which the secret you attempted to commit is. Share a screenshot of that fail message in a comment on this PR.
- Be sure to approve this PR if all went well. It helps determine who was successful and who might need some assistance. We shall merge the PR when all reviewers were successful and approved.

#### What are the relevant tickets?
- [PLAT-1078](https://airqoteam.atlassian.net/browse/PLAT-1078)

#### Screenshots (optional)
